### PR TITLE
3.get api/topics

### PR DESCRIPTION
**Pull request by SamirabintABS**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    it('GET: 200 responds with an array of all topic objects', () => {
+        return request(app)
+            .get("/api/topics")
+            .expect(200)
+            .then(({ body }) => {
+                const { topics } = body;
+                expect(topics).toHaveLength(3);
+            })
+    });
+    it('GET 200 responds with an array of topic objects, each one should have slug and description properties', () => {
```
Comment (NRMSMITH) [inline __tests__/app.test.js]: this could be part of the above test!

File: app.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+const { getTopics } = require('./controllers/app.controller');
+
+const app = express();
+
+app.get("/api/topics", getTopics);
+
+app.all("*", (req, res, next) => {
+    res.status(404).send({ msg: 'Path not found' })
+})
+
```
Comment (NRMSMITH) [inline app.js:11]: I would consider handling 500 errors here too :)

File: controllers/app.controller.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+const { fetchTopics } = require("../models/app.model")
+
+exports.getTopics = (req, res) => {
+    fetchTopics().then((topics) => {
+        res.status(200).send({ topics })
+    })
+}
```
Comment (NRMSMITH) [inline controllers/app.controller.js:7]: consider having a .catch(next) as you will need it later :)

**Review by NRMSMITH — APPROVED** [review]
🚀 well done Samira! A couple of notes!

**Comment by SamirabintABS** [comment]
I have tried to resolve the notes in the most recent commit

**Comment by NRMSMITH** [comment]
Merge it!! 💃

**Pull request merged**